doc/rados/troubleshooting/troubleshooting-pg.rst

```diff
@@ -452,9 +463,28 @@ some predefined criteria. But this does not always work. For example, the stored
 data digest could be missing, and the calculated digest will be ignored when
 choosing the authoritative copies. So, please use the above command with caution.
 
-If ``read_error`` is listed in the ``errors`` attribute of a shard, the
-inconsistency is likely due to disk errors. You might want to check your disk
-used by that OSD.
+If you are able to identify the `good` copy by inspecting it, it is advisable to
+use the ``rados`` command or the RADOS APIs to fix the inconsistencies manually
+instead. In the above example, if the ``selected_object_info`` is hard to decipher
+or does not make sense at all, you will need to download the payload of the shards
+and inspect them manually::
+
+  rados -p rbd --force repair-get foo 0 14 foo.osd-0
+
+This command fetches the replica of ``foo`` to the local file ``foo.osd.0``
+from osd.0 at epoch 14. If you are happy with this replica after looking at
+it, you might want to overwrite bad replicas with good ones::
+
+  rados -p rbd repair-copy foo 2 14 1
```

Comment:
the version assert to prevent us from overwriting the changed object with the old copy.